lib/confluence/api_velocity_data.es6.js

```diff
@@ -26,12 +26,16 @@ foam.CLASS({
       final: true,
```

Comment:
Not urgent: Could this belong to a base class that uses an `expression` to look up the name from the browser object already stored in the metric object?